query-engine/connector-test-kit-rs/query-tests-setup/src/connector_tag/mod.rs

```diff
@@ -378,12 +383,12 @@ pub(crate) fn should_run(
 
     let exclusions = exclude
         .iter()
-        .filter_map(|c| ConnectorVersion::try_from(*c).ok())
+        .map(|c| ConnectorVersion::try_from(*c).unwrap())
         .collect::<Vec<_>>();
 
     let inclusions = only
         .iter()
-        .filter_map(|c| ConnectorVersion::try_from(*c).ok())
+        .map(|c| ConnectorVersion::try_from(*c).unwrap())
         .collect::<Vec<_>>();
```

Comment:
Unrelated change to make sure that when a version is specified for the `only` / `exclude` attributes in the test suite, if that version is incorrectly specified, it errors instead of silently ignoring it.